dart/test/sentry_client_test.dart

```diff
@@ -373,6 +374,7 @@ void main() {
     );
 
     setUp(() {
+      fixture = Fixture();
       options = SentryOptions(dsn: fakeDsn);
       options.transport = MockTransport();
       scope = Scope(options)
```

Comment:
if those things are used across all the tests, they should be part of the fixture too